### PR TITLE
README Update

### PR DESCRIPTION
**Pull request by Darshini-Parikh93**
- Added openssl command for generating public key hash
- Added default hashlist in demo README

Signed-off-by: Darshini Parikh <darshini.k.parikh@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,11 @@ The Rendezvous Service can block the whole Ownership Voucher by adding GUID into
...
 * Create Redis Hash **OP_KEYS_WHITELIST**
-* Calculate SHA256 hash of an ownership voucher public key (upper case, example: "0734FAC43DBE455D531930B6A8E024043356541BFFCC7A250E417EC38E217725")
+* Calculate SHA256 hash of an ownership voucher public key using command <br/>
+(upper case, example: "0734FAC43DBE455D531930B6A8E024043356541BFFCC7A250E417EC38E217725")
+```
+$ openssl x509 -in cert.pem -pubkey -noout | openssl enc -base64 -d > example_whitelist_publickey.der 
+```
 * Add the calculated hash to the **OP_KEYS_WHITELIST** with the calculated hash as a key and "1" as a value, example:
 ```
 $ cat ./trustmanagement/example_whitelist_publickey.der | openssl dgst -sha256 | awk '/s/{print toupper($2)}'
```
Comment (trbehera) [inline README.md:72]: This line can also be updated to remove the folder name "trustmanagement' from example command.

File: README.md [hunk 2]
```diff
@@ -62,7 +62,11 @@ The Rendezvous Service can block the whole Ownership Voucher by adding GUID into
 
 **Prepare ownership voucher keys whitelist**:
 * Create Redis Hash **OP_KEYS_WHITELIST**
-* Calculate SHA256 hash of an ownership voucher public key (upper case, example: "0734FAC43DBE455D531930B6A8E024043356541BFFCC7A250E417EC38E217725")
+* Calculate SHA256 hash of an ownership voucher public key using command <br/>
+(upper case, example: "0734FAC43DBE455D531930B6A8E024043356541BFFCC7A250E417EC38E217725")
+```
+$ openssl x509 -in cert.pem -pubkey -noout | openssl enc -base64 -d > example_whitelist_publickey.der
```
Comment (trbehera) [inline README.md:68]: Please specify explicitly that the input file cert.pem is an X509 certificate.

File: demo/README.md [hunk 3]
```diff
@@ -25,7 +25,29 @@ If you are working from the Release Package, these files have already been copie
 If you are working with source code, build the project and it will copy the required dependencies to the demo folder.
 
 ## Create Java keystore files
-See instructions in the [Secure Device Onboard Rendezvous Service README](https://github.com/secure-device-onboard/rendezvous-service/#generate-keystores). Once the keystore and truststore files are created, update docker-compose.yml to reflect the file name, path and password.  The default configured keystore is /certs/rendezvous-keystore.jks and the default configured truststore is /certs/rendezvous-truststore.jks. Default passwords for both: 123456
+The example IOT Platform SDK, Supply Chain Toolkit - Manufacturer and Supply Chain Toolkit - Reseller
+public key hash are whitelisted in the redis docker deployment. Please find the list of whitelisted hash and
+its corresponding SDO component.
```
Comment (trbehera) [inline demo/README.md:30]: You may consider to rephrase this as below.
"The hashes for the default public keys present in iot-platform-sdk and supply-chain-tools are listed in following table."

File: README.md (hunk 1 above)
Comment (Darshini-Parikh93) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Darshini-Parikh93) [reply README.md:69]: Done

File: demo/README.md (hunk 3 above)
Comment (Darshini-Parikh93) [reply demo/README.md]: Done
